### PR TITLE
add easy-coding-standard

### PR DESCRIPTION
**Pull request by fbuchlak**

### CODE REVIEW

File: packages/easy-coding-standard/package.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+languages:
+  - PHP
+categories:
+  - Formatter
+
+source:
+  id: pkg:composer/symplify/easy-coding-standard@12.0.8
+
+bin:
+  ecs: composer:bin/ecs
```
Comment (williamboman) [inline packages/easy-coding-standard/package.yaml:16]: The `composer:` prefix will inform clients (i.e. mason.nvim) to locate the executable according to platform-local rules, so no need to provide a path.

```suggestion
  ecs: composer:ecs
```

**Review by williamboman — APPROVED** [review]
Thanks!

File: packages/easy-coding-standard/package.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+name: easy-coding-standard
+description: Use Coding Standard with 0-knowledge of PHP-CS-Fixer and PHP_CodeSniffer.
+homepage: https://github.com/easy-coding-standard/easy-coding-standard
+licenses:
+  - MIT
+languages:
+  - PHP
+categories:
+  - Formatter
```
Comment (williamboman) [inline packages/easy-coding-standard/package.yaml:10]:
```suggestion
  - Formatter
  - Linter
```

**Comment by fbuchlak** [comment]
Might need to rerun tests. I'm not sure if I can do anything.

**Comment by williamboman** [comment]
> Might need to rerun tests. I'm not sure if I can do anything.

Yeah I reran it some time ago and it failed again, not sure why. Let's try again

**Pull request closed without merging**